components/Footer/footer.js

```diff
@@ -20,22 +20,22 @@ function Footer() {
   ];
   return (
     <div className="container">
-          <div className="w-full flex justify-between items-center p-4">
-            <img src="/img/logo.png" className="w-[150px]" />
-      <div className="mt-2 text-[14px] text-gray underline">
-        <a
-        href="https://github.com/asyncapi/.github/blob/master/CODE_OF_CONDUCT.md"
-        target="_blank"
-        rel="noreferrer"
-        className="hover:text-sky-500 duration-200 ease-in-out"
-        >
-              Code of Conduct
-            </a>
-          </div>
-      <div>
-      </div>
+      <div className="w-full flex justify-between items-center p-4 sm:flex-col sm:gap-3">
+        <img src="/img/logo.png" className="w-[150px]" />
+        <div className="mt-2 text-[14px] text-gray-100 underline">
+          <a
+            href="https://github.com/asyncapi/.github/blob/master/CODE_OF_CONDUCT.md"
+            target="_blank"
+            rel="noreferrer"
+            className="hover:text-sky-500 duration-200 ease-in-out"
+          >
+            Code of Conduct
+          </a>
+        </div>
+        <div>
+        </div>
```

Comment:
suggestion: remove the indentation changes.

Reply:
@RamGoel PTAL!